core/src/main/java/tc/oc/pgm/spawner/SpawnerDefinition.java

```diff
@@ -10,24 +10,25 @@
 @FeatureInfo(name = "spawner")
 public class SpawnerDefinition implements FeatureDefinition {
 
+  public final String numericId;
   public final Region spawnRegion;
   public final Region playerRegion;
   public final int maxEntities;
   public final Duration minDelay, maxDelay, delay;
   public final List<Spawnable> objects;
   public final Filter playerFilter;
-  public final int numericID;
 
   public SpawnerDefinition(
+      String numericId,
```

Comment:
```suggestion
      String id,
```
tf does numeric id mean, this is plain and simply, id.

extend SelfIdentifyingFeatureDefintion and pass it up to the parent, do not store it in SpawnerDefintion.